### PR TITLE
Fix window position in SDL 2 for Windows

### PR DESCRIPTION
**Pull request by ihhub**
It's some kind of hack but at least it works

relates to #232

### CODE REVIEW

**Comment by andrey-starodubtsev** [comment]
seems a little hacky indeed, but mustn't break anything

I cannot reproduce on my Win setup though. Have you changed default config in some way?

**Comment by ihhub** [comment]
Nope, I haven't changed the config in any way. It's a default behavior for my system.

File: src/engine/display.cpp [hunk 1]
```diff
@@ -62,7 +62,8 @@ void Display::SetVideoMode(int w, int h, bool fullscreen)
     if(window)
         SDL_DestroyWindow(window);
 
-    window = SDL_CreateWindow("", 0, 0, w, h, flags);
```
Comment (andrey-starodubtsev) [inline src/engine/display.cpp]: Found better way to handle this from https://wiki.libsdl.org/SDL_CreateWindow: use SDL_WINDOWPOS_CENTERED.

Comment (ihhub) [reply src/engine/display.cpp]: I was thinking the same...

**Pull request merged**